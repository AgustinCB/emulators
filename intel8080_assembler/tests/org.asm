ORG 100H
MVI A, 10