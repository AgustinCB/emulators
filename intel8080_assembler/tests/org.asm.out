0100 MVI A,#$0a
